content/22.vaccines.md

```diff
@@ -178,6 +178,33 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
+Phase 1/2 of clinical trial was registered on March 27, 2020 [@clinicaltrials:NCT04324606] and enrolled participants between April 23 and May 21, 2020. 
+This single-blind, randomised controlled trial was carried out on 1077 participants: 543 received  ChAdOx1 nCoV-19 and 534 meningococcal vacine MenACWY. 
+The median age of trial participants was 35 years (IQR 28–44), the cohort included predominately white (979, 90.9%) participants, out of which 536 were female (49.8%) and 541 were males (50.2%). 
+The vaccine showed safe profile, with spike-specific  T-cell peaking  on  day  14  and anti-spike IgG responses  rising  by  day  28 and remaining elevated to day 56.
+Some systemic adverse local reactions included pain, feeling feverish, chills, muscle ache, headache, and malaise  [@doi:10.1016/S0140-6736(20)31604-4].
+The Phase II/III clinical trial was initiated in the UK and enrolled 560 participants of different age between May 30 and Aug 8, 2020 as a single-blinded randomized trial [@clinicaltrials:NCT04400838]. 
+The main finding were safety of the vaccine, decreased anti-spike IgG responses at day 28 with age, but similar level of antibody titres after receiving a booster dosage [@doi:10.1016/S0140-6736(20)32466-1].
+
+
+Phase II/III clinical trial was carried out as 4 independent locations: in the UK  [@clinicaltrials:NCT04400838], Brasil [@clinicaltrials:NCT04536051] and South Africa  [@clinicaltrials:NCT04444674] and finally in the United States in August 2020 [@clinicaltrials:NCT04516746; @url:https://s3.amazonaws.com/ctr-med-7111/D8110C00001/52bec400-80f6-4c1b-8791-0483923d0867/c8070a4e-6a9d-46f9-8c32-cece903592b9/D8110C00001_CSP-v2.pdf]. 
+In September 2020, the global trials had to be shortly paused due to a serious adverse neurological reaction in some of the participants [@doi:10.1038/d41586-020-02633-6].
+
+The interim results of Phase III were announced in a press release on November 23, 2020 [@url:https://www.astrazeneca.com/content/astraz/media-centre/press-releases/2020/azd1222hlr.html] and published in Lancet on December 9, 2020 [@doi:10.1016/S0140-6736(20)32661-1; @url:https://www.astrazeneca.com/content/astraz/media-centre/press-releases/2020/azd1222-oxford-phase-iii-trials-interim-analysis-results-published-in-the-lancet.html]. 
+The studies were met with skepticism, due to multiple inonsistencies in their design, including the errorneous dosing regimen in which multiple of 11,636 participants received half of the initial dose eventually leading to increased vaccine efficacy (90% vs 62.1% for two full doses protocol), using multiple testing sites and different placebos (meningococcal conjugate vaccine or saline) [@url:https://www.businessinsider.com/astrazenecas-covid-19-vaccine-trials-criticised-2020-11; @doi:10.1136/bmj.m4777; @doi:10.1016/S0140-6736(20)32623-4].
```

Comment:
```suggestion
The studies were met with skepticism, due to multiple inconsistencies in their design.
These include the erroneous dosing regimen in which many of the 11,636 participants received half of the initial intended dose eventually leading to increased vaccine efficacy (90% vs 62.1% for two full doses protocol), using multiple testing sites and different placebos (meningococcal conjugate vaccine or saline) [@url:https://www.businessinsider.com/astrazenecas-covid-19-vaccine-trials-criticised-2020-11; @doi:10.1136/bmj.m4777; @doi:10.1016/S0140-6736(20)32623-4].
```

Reply:
In have added wording suggestions, but I agree with @ajlee21, would a lower dose not lead to lower efficacy . . . Something not clear here